src/index.css

```
@@ -1,50 +1,368 @@
-  text-decoration: inherit;
-}
-a:hover {
-  color: #535bf2;
+body.light {
+  background: #f8fafc;
+  color: #0f172a;
 }
 
 body {
-  margin: 0;
+  font-family: "Inter", sans-serif;
```

Stylelint (17.7.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)